mlir/python/mlir/_mlir_libs/__init__.py

```diff
@@ -56,14 +56,28 @@ def get_include_dirs() -> Sequence[str]:
 #
 # This facility allows downstreams to customize Context creation to their
 # needs.
+
+_dialect_registry = None
```

Comment:
Document please?

Reply:
My first todo for tomorrow is to update the python docs with all the stuff from the last ~6 months 😄.